api/v1alpha1/edgedevice_types.go

```diff
@@ -34,9 +34,24 @@ type EdgeDeviceSpec struct {
 	// RequestTime is the time of device registration request
 	RequestTime *metav1.Time `json:"requestTime,omitempty"`
 
-	Heartbeat *HeartbeatConfiguration `json:"heartbeat,omitempty"`
-	Storage   *Storage                `json:"storage,omitempty"`
-	Metrics   *MetricsConfiguration   `json:"metrics,omitempty"`
+	Heartbeat     *HeartbeatConfiguration         `json:"heartbeat,omitempty"`
+	Storage       *Storage                        `json:"storage,omitempty"`
+	Metrics       *MetricsConfiguration           `json:"metrics,omitempty"`
+	LogCollection map[string]*LogCollectionConfig `json:"log_collection,omitempty"`
```

Comment:
Should we be consistent with naming and have here json:"logCollection"

Reply:
You're right. the convention is camelCase.

Reply:
Created https://github.com/project-flotta/flotta-operator/pull/124